src/main/java/org/folio/search/service/IndexService.java

```diff
@@ -92,6 +95,12 @@ public void createIndexIfNotExist(String resourceName, String tenantId) {
     var index = getElasticsearchIndexName(resourceName, tenantId);
     if (!indexRepository.indexExists(index)) {
       createIndex(resourceName, tenantId);
+    } else if (log.isInfoEnabled()) {
+      log.info("Index already exists [index={}]", index);
```

Comment:
Is this needed? Method name is `createIndexIfNotExist`

Reply:
The log statement is removed.